src/util/config/build.js

```diff
@@ -61,6 +76,16 @@ if (asset_server) {
 
 config.set('isProd', config.get('env') === 'production');
 config.set('isDev', config.get('env') === 'development');
+
+const assetConf = config.get('asset_server');
```

Comment:
This is a little duplication, but I think I kind of prefer it to creating another thing to import.